test/integration/resultsPagePlace.js

```diff
@@ -114,7 +114,7 @@ describe('The place results page', () => {
         const $ = cheerio.load(res.text);
         expectSearchAgainPage($);
         expect($('title').text()).to.equal('Find a pharmacy - We can\'t find the postcode \'\' - NHS.UK');
-
+        expect($('.form-label-bold').text()).to.equal('Enter a town, city or postcode in England');
```

Comment:
Could this not be part of https://github.com/nhsuk/connecting-to-services/pull/401/files#diff-92428b787dba582ae2e87d8a8fabc1e6R105 ?